src/System.Reflection/tests/AssemblyTests.cs

```diff
@@ -34,7 +34,7 @@ public class AssemblyTests : FileCleanupTestBase
 
         public AssemblyTests() 
         {
-            // Assembly.Location not supported (properly) on uapaot.
+            // Assembly.Location not supported (properly) on UWP.
```

Comment:
Is this comment valid?

Reply:
```suggestion
            // Assembly.Location does not return the file path for single-file deployment targets
```

`Assembly.Location` does not return the file path for single-file deployment targets where the individual .dlls are not present on disk as files. `uapaot` was an example of such deployment target. The single-file deployment targets  are being worked on for .NET Core as well.